src/amber/controller/base.cr

```diff
@@ -26,5 +26,10 @@ module Amber::Controller
     def controller_name
       self.class.name.gsub(/Controller/i, "")
     end
+
+    # If static resource is not found then raise an exception
+    def index
```

Comment:
Still my concern with this is that all controllers inherit it. My initial comment on this suggested moving it to a new controller in amber/controller/ until we remove it. That way it doesn't clutter the project and it gives us an incentive to remove the need for it so it doesn't clutter amber core either.

`Amber::Controller::Static` is what I would suggest calling it for now. What are your thoughts on that?

Reply:
oooh, right.  i see. ok.